### PR TITLE
[am] Color edges based on AM status

### PR DESCRIPTION
**Pull request by JDevlieghere**
This changes the color of the edges to green or red based on the status
of the automerger. I've also made the nodes gray as the light blue
didn't work very well with the colored edges.

### CODE REVIEW

**Comment by JDevlieghere** [comment]
@swift-ci test

File: git_apple_llvm/am/am_graph.py [hunk 1]
```diff
@@ -11,6 +13,23 @@
     pass
 
 
+def is_blocked(config: AMTargetBranchConfig, remote: str = 'origin'):
+    if not config.common_ancestor:
+        return False
+    commit_log_output = git_output(
+        'log',
+        '--first-parent',
+        '--pretty=format:%H %cd', '--no-patch',
```
Comment (hyp) [inline git_apple_llvm/am/am_graph.py]: If you do format '%H', you won't need to do the second split to find the hash, as you'll be listing hashes only without the dates.

**Comment by JDevlieghere** [comment]
@swift-ci test

**Comment by JDevlieghere** [comment]
@swift-ci test

**Comment by hyp** [comment]
@JDevlieghere based on the discussion of the new merge idea from @dexonsmith and @fredriss, we won't actually need to determine if the secondary edge is merged by a blocking edge. I would suggest reworking the PR to color differently if the status is different, based on the redis state. Let me create a PR that puts up the redis state enum code here.

**Comment by JDevlieghere** [comment]
Sounds good!

**Comment by JDevlieghere** [comment]
@swift-ci please test

**Comment by JDevlieghere** [comment]
@swift-ci please test

**Pull request merged**